### ISSUE TITLE
Color tint of background imagery changes on panning

### ISSUE THREAD

**Issue by habi** [bryceco/GoMap#527]
While editing around [this parking lot](https://www.openstreetmap.org/way/595737405/) and panning around a bit I noticed that the tint of the background imagery (swisstopo SWISSIMAGE) changes noticeably.
 
https://user-images.githubusercontent.com/1651235/119396340-056cbb80-bcd5-11eb-9d55-71031f2364e5.mov

I *think* it might be related to the outline of the nearby lake, but was not able to figure out exactly...

Here's [one tile of the background layer on the parking lot](http://wms.geo.admin.ch/?LAYERS=ch.swisstopo.swissimage&STYLES=default&FORMAT=image/jpeg&CRS=EPSG:3857&WIDTH=512&HEIGHT=512&BBOX=849138.9472200,5899792.0281913,849215.3842483,5899868.4652196&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap) and here's [one adjacent in the 'harbour'](http://wms.geo.admin.ch/?LAYERS=ch.swisstopo.swissimage&STYLES=default&FORMAT=image/jpeg&CRS=EPSG:3857&WIDTH=512&HEIGHT=512&BBOX=849215.3842483,5899792.0281913,849291.8212766,5899868.4652196&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap) (found through JOSM)

**Comment by habi** [comment]
BTW: Go Map!! (TestFlight Version 2.2.1 (2210) on iOS 14.5.1 on an iPhone 12 mini

**Comment by bryceco** [comment]
Yeah, that's because the lake's boundary is a coastline, which means there's no easy way to determine whether the current view is land or water, and it's guessing incorrectly. It's definitely something I'd like to fix.

**Comment by habi** [comment]
> It's definitely something I'd like to fix.

(As always,) thanks for the quick feedback, the intention to look into it and for a great app!
